connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/connect/client/util/ConnectFunSuite.scala

```diff
@@ -16,9 +16,43 @@
  */
 package org.apache.spark.sql.connect.client.util
 
+import java.nio.file.Path
+
 import org.scalatest.funsuite.AnyFunSuite // scalastyle:ignore funsuite
 
 /**
  * The basic testsuite the client tests should extend from.
  */
-trait ConnectFunSuite extends AnyFunSuite {} // scalastyle:ignore funsuite
+trait ConnectFunSuite extends AnyFunSuite { // scalastyle:ignore funsuite
+
+  // Borrowed from SparkFunSuite
+  protected def getWorkspaceFilePath(first: String, more: String*): Path = {
```

Comment:
We have similar code in the client test util. Shall we just keep one of this method?

Reply:
Which file is that? I moved it out of `PlanGenerationSuite` and put it here

Reply:
My bad, only `sparkHome` in `IntegrationTestUtils#sparkHome`
We can also clean this up in another PR.